### PR TITLE
fix metric documentation

### PR DESCRIPTION
**Pull request by jsteemann**
### Scope & Purpose

Fix an inaccurate description for `agency_callback_number` metric.

- [x] :hankey: Bugfix
- [ ] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [x] Backport for 3.11: https://github.com/arangodb/arangodb/pull/20544
  - [x] Backport for 3.10: https://github.com/arangodb/arangodb/pull/20545

#### Related Information

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

File: Documentation/Metrics/rocksdb_cache_ongoing_tasks.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+complexity: advanced
+exposedBy:
+  - dbserver
+  - agent
+  - single
+description: |
+  Total number of currently ongoing tasks that are scheduled by the
+  in-memory edge cache subsystem. The value reported by this metric
+  should not exceed the value of the startup option 
+  `--cache.max-parallel-tasks`.
```
Comment (hkernbach) [inline Documentation/Metrics/rocksdb_cache_ongoing_tasks.yaml:18]: Does this belong here into this PR?

**Review by hkernbach — COMMENTED** [review]
LGTM, except the file where I am not sure if it does belong here into this PR.

**Comment by jsteemann** [comment]
> LGTM, except the file where I am not sure if it does belong here into this PR.

Very good catch, I didn't pay close enough attention here :facepalm:

**Review by hkernbach — APPROVED** [review]

**Review by ansoboleva — APPROVED** [review]
lgtm

**Pull request merged**